.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+name: Create Release and Publish
+
+on:
+  push:
+    tags:
+      - 'dcap-artifact-retrieval_v*'
+      - 'dcap-retrieve-pckid_v*'
```

Comment:
Should we also do this for the other executables to remain consistent?

Reply:
I'm mainly thinking of `sgx-detect` as that's something users do execute repeatedly. But all SGX related executables could be in scope such as `sgxs-info`, ...